src/components/Booking/Booking.jsx

```diff
@@ -1,14 +1,24 @@
 import dayjs from "dayjs";
+import { useState } from "react";
```

Comment:
I didn't know you can import something like this. What does this mean?

Reply:
If there is something you have not learned in class, it might be a good idea to pop in some comments explaining your code!